File: Plugins/Aspose_Imaging_Java_for_Ruby/lib/asposeimagingjava/images/rotateandflipimage.rb
module Asposeimagingjava
  module RotateAndFlipImage
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'

        # Load an existing image (of type bmp) in an instance of Image class
        image = Rjb::import('com.aspose.imaging.Image').load(data_dir + "test.jpg")

        image.rotateFlip(Rjb::import('com.aspose.imaging.RotateFlipType').Rotate270FlipNone)

        # Save the image to disk
        image.save(data_dir + "RotateFlip.jpg")

        # Display Status.
        puts "RotateFlip image successfully!"
    end
  end
end

File: Plugins/Aspose_Imaging_Java_for_PHP/src/aspose/imaging/ManagingRasterFormats/RotateAndFlipImage.php
<?php
namespace Aspose\Imaging\ManagingRasterFormats;

use com\aspose\imaging\Image as Image;
use com\aspose\imaging\RotateFlipType as RotateFlipType;
class RotateAndFlipImage{

    public static function run($dataDir=null){

        # Load an existing image (of type bmp) in an instance of Image class
        $image=new Image();
        $image = $image->load($dataDir . "test.jpg");

        $rotateFlipType=new RotateFlipType();
        $image->rotateFlip($rotateFlipType->Rotate270FlipNone);

        # Save the image to disk
        $image->save($dataDir . "RotateFlip.jpg");

        # Display Status.
        print "RotateFlip image successfully!".PHP_EOL;
    }
}
?>